addons/hr_holidays/models/hr_leave.py

```diff
@@ -260,6 +262,34 @@ def _auto_init(self):
                            self._table, ['date_to', 'date_from'])
         return res
 
+    @api.depends_context('uid')
+    def _compute_description(self):
+        self.check_access_rights('read')
+        self.check_access_rule('read')
+
+        is_officer = self.user_has_groups('hr_holidays.group_hr_holidays_user')
+        user_id = self.env.user.id
+
+        for leave in self:
+            if is_officer or leave.user_id.id == user_id or (leave.manager_id and leave.manager_id.id == user_id):
+                leave.name = leave.sudo().private_name
+            else:
+                leave.name = '*****'
+
+    def _inverse_description(self):
+        for leave in self:
+            leave.sudo().private_name = leave.name
```

Comment:
Does this mean everyone that can write on the field `name` can effectively also write on `private_name` ?